src/main/java/com/choelinska/horrorlibz/services/impl/UserServiceImpl.java

```diff
@@ -25,12 +25,24 @@ public UserServiceImpl(UserRepository userRepository, ModelMapper modelMapper){
 
     @Override
     public void saveUser(UserTo user) {
-        userRepository.save(mapToEntity(user));
+        userRepository.save(this.mapToEntity(user));
+    }
+
+    @Override
+    public List<UserEntity> getSortedByName(){
+        return this.userRepository.findAll(sortByNameDesc());
+    }
+
+    @Override
+    public List<UserEntity> getUsers(){
+        return this.userRepository.findAll();
+    }
+
+    private Sort sortByNameDesc() {
+        return Sort.by(Sort.Direction.DESC, "username");
     }
 
     private UserEntity mapToEntity(UserTo user){
-        return modelMapper.map(user, UserEntity.class);
+        return this.modelMapper.map(user, UserEntity.class);
     }
-    private UserTo convertToDto(UserEntity userEntity) {
```

Comment:
You need to keep this function if the mapping to TO is going to function correctly.